src/utils/xcmApi/transfer.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  CollateralUnit,
+  DefaultTransactionAPI
+} from '@interlay/interbtc-api';
+import {
+  Currency,
+  MonetaryAmount
+} from '@interlay/monetary-js';
+import { web3FromAddress } from '@polkadot/extension-dapp';
+import { AddressOrPair } from '@polkadot/api/types';
+import { ApiPromise } from '@polkadot/api';
+
+import { RELAYCHAIN_ID } from '../../constants';
+import { createRelayChainApi } from './createRelayChainApi';
+
+type XCMTransferAmount = MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>;
+
+const createDest = (api: ApiPromise) => {
+  const x1 = api.createType('XcmV1Junction', { parachain: RELAYCHAIN_ID });
+  const interior = api.createType('XcmV1MultilocationJunctions', { x1 });
+  const v1 = api.createType('XcmV1MultiLocation', { parents: 0, interior });
```

Comment:
Imo the code was easier to read when it wasn't split out into the separate `x1`, `interior` and `v1` variables, but maybe that's just what I'm used to